### ISSUE TITLE
Bug render value when double tap continuous

### ISSUE THREAD

**Issue by webserveis** [KasualBusiness/MaterialNumberPicker#17]
If you touch repeatedly on the selected value , often it makes a small jump

**Comment by StephenVinouze** [comment]
Yes, you're right, seems to happen just after moving the wheel, then it never occurs once the item is properly set in the box.
Can't really do anything about it though... This library is merely a wrapper that do lots of reflection to tweak its erratic behavior :/
